vocabularies.md

```diff
@@ -0,0 +1,283 @@
+        "type": "vocabularySuggestion"
+      }]
+  },
+  "_links": {
+    "first": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.type&page=0&size=2"
+    },
+    "self": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.type&size=2"
+    },
+    "last": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.type&page=0&size=2"
+    }
+  },
+  "page": {
+    "size": 2,
+    "totalElements": 2,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+sample for a hierarchical authority  (srsc): 
+/server/api/submission/vocabularies/srsc/entries?metadata=dc.subject&filter=Research&size=2
+
+```json
+{
+  "_embedded": {
+    "vocabularyEntries": [
+      {
+        "display": "Family research",
+        "value": "Research Subject Categories::SOCIAL SCIENCES::Social sciences::Social work::Family research",
+        "otherInformation": {
+          "id": "VR131402",
+          "parent": "Research Subject Categories::SOCIAL SCIENCES::Social sciences::Social work",
+          "hasChildren": "false",
+          "note": "Familjeforskning"
+        },
+        "type": "vocabularyEntry",
+        "_links": {
+          "vocabularyEntryDetail": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularyEntryDetails/srsc:VR131402"
+          }
+        }
+      },
+      {
+        "display": "Youth research",
+        "value": "Research Subject Categories::SOCIAL SCIENCES::Social sciences::Social work::Youth research",
+        "otherInformation": {
+          "id": "VR131403",
+          "parent": "Research Subject Categories::SOCIAL SCIENCES::Social sciences::Social work",
+          "hasChildren": "false",
+          "note": "Ungdomsforskning"
+        },
+        "type": "vocabularyEntry",
+        "_links": {
+          "vocabularyEntryDetail": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularyEntryDetails/srsc:VR131403"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "first": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.subject&filter=Research&size=2&page=0"
+    },
+    "self": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.subject&filter=Research&size=2"
+    },
+    "next": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.subject&filter=Research&size=2&page=1"
+    },
+    "last": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries?metadata=dc.subject&filter=Research&size=2&page=12"
+    }
+  },
+  "page": {
+    "size": 2,
+    "totalElements": 26,
+    "totalPages": 13,
+    "number": 0
+  }
+}
+```
+
```

Comment:
Can an example for an authority (e.g. the solr authority) be included here as well? The resulting format was different if I remember correctly from the wiki

Reply:
done

Reply:
thanks